@xen-orchestra/web-core/lib/enabled-state/VtsEnabledState.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <UiInfo :accent="state.accent">
+    {{ state.label }}
+  </UiInfo>
+</template>
+
+<script setup lang="ts">
+import UiInfo, { type InfoAccent } from '@core/components/ui/info/UiInfo.vue'
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+type StatesMap = Record<'enabled' | 'disabled', { label: string; accent: InfoAccent }>
+
+const { enabled } = defineProps<{
+  enabled: boolean
+}>()
+
+const { t } = useI18n()
+
+const statesMap: StatesMap = {
+  enabled: { label: t('enabled'), accent: 'success' },
+  disabled: { label: t('disabled'), accent: 'muted' },
+}
+
+const state = computed(() => (enabled ? statesMap.enabled : statesMap.disabled))
```

Comment:
Maybe we can simplify this like that ? 

```suggestion
const state = computed<StatesMap>(() => (
  enabled
    ? { label: t('enabled'), accent: 'success' }
    : { label: t('disabled'), accent: 'muted' }
))
```